Resources/Prototypes/_EE/Entities/NanoTrasen/Modsuits/leader-responsory.yml

```
@@ -0,0 +1,181 @@
+          - state: equipped-FEET-sealed
+
+- type: entity
+  id: SuitStorageResponsoryLeader
+  parent: SuitStorageBase
+  suffix: Leader Responsory Modsuit
+  components:
+    - type: StorageFill
+      contents:
+        - id: OxygenTankFilled
+        - id: ClothingModsuitResponsoryLeader
+        - id: ClothingMaskBreath
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)